CNCF website guidelines checklist into sandbox on-boarding template (#294)
cncf/sandbox: .github/ISSUE_TEMPLATE/project-onboarding.md

- [ ] Accept the invite to join the CNCF GitHub Enterprise account. We'll then add `thelinuxfoundation` as an organization owner to ensure neutral hosting of your project.
- [ ] Migrate your Slack channels (if any) to the [Kubernetes or CNCF Slack workspace](https://slack.com/help/articles/217872578-Import-data-from-one-Slack-workspace-to-another). CNCF staff can help.
- [ ] Submit a [pull request](https://github.com/cncf/landscape/pulls) to add your project as a Sandbox project to the [Cloud Native Landscape](https://landscape.cncf.io) by updating [landscape.yml](https://github.com/cncf/landscape/blob/master/landscape.yml) following these [instructions](https://github.com/cncf/landscape2/blob/main/docs/config/data.yml).
- [ ] Transfer [your domain(s) to the CNCF](https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/create/63). The "LF Stakeholder email" is <projects@cncf.io>.
- [ ] Transfer any [trademark and logo assets to the Linux Foundation](https://github.com/cncf/foundation/tree/main/agreements).
- [ ] Submit a [pull request](https://github.com/cncf/artwork) with your existing artwork.  

> _Note_: If you don't have artwork, CNCF can help design some.

Author: Should we include more info about our design services and how to access them?

> - [ ] Transfer website analytics to <projects@cncf.io>, if applicable. CNCF staff can help.

Author: If applicable as not all projects will have analytics, and for those that do, it may not be Google Analytics so another transfer method may need to be used.

- [ ] Submit a [pull request](https://github.com/cncf/artwork) with your existing artwork.  
_Note_: If you don't have artwork, CNCF can help design some.
- [ ] Transfer website analytics to <projects@cncf.io>, if applicable. CNCF staff can help.

## Update and document

- [ ] Ensure that [DCO](https://github.com/apps/dco) (preferred) or [CLA](https://github.com/cncf/cla) are enabled for all GitHub repositories of the project.
- [ ] Ensure that that the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md) (or your adopted version of it) are explicitly referenced at the project's `README` on GitHub.

> - [ ] Ensure LF footer is on your website and [guidelines](https://github.com/cncf/foundation/blob/main/website-guidelines.md) are followed (if your project doesn't have a dedicated website, please adopt those guidelines for the `README` file).

Author: moved to the _Websites_ section.

## Websites

_Note_, not all of these are applicable to all projects

- [ ] Ensure website is
  - [ ] [Hosted in an open source repo](https://github.com/cncf/techdocs/blob/main/docs/repo-setup.md)
  - [ ] Hosted in the same organization as the main project
  - [ ] Setup with [DCO](https://github.com/apps/dco) or CLA (DCO recommended)
[CNCF's IP policy](https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy)
requires all projects to use either CLA (Contributor License Agreements)
or [DCO (Developer Certificate of Origin)](https://github.com/apps/dco).
Unless there's a strong necessity to use CLA, we encourage projects to

> use DCO as it's easier to setup and use.

Author: This may duplicate some instructions above, particularly line 42 "Ensure that [DCO](https://github.com/apps/dco) (preferred) or [CLA](https://github.com/cncf/cla) are enabled for all GitHub repositories of the project."

We may want to drop it from here (though we may want to provide some of this info above)

- [ ] Ensure references to any origin companies are correct  
  _Note_: It is OK to say that, e.g., “Prometheus was originally created by
  Soundcloud” or “Kubernetes builds upon 15 years of experience of running
  production workloads at Google,” but the origin company should not otherwise
  be referred to on the project homepage.

> - [ ] No links or forms for capturing enterprise support leads should be present  
>   _Note_: It is fine to have an enterprise support, commercial partners, or similar page.

Author: ```suggestion
- [ ] Remove any links or forms for capturing enterprise support leads  
  _Note_: It is fine to have an enterprise support, commercial partners, or similar page.
```

  - [ ] If page is present, the companies list is alphabetized or randomized on load.
  - [ ] If page is present, the vetting of companies listed is complete
    _Note_: Projects are welcome to outsource this vetting to CNCF staff if this
    becomes a burden.

> - [ ] Links to companies offering support go to a page that at least mentions 
>   support of the project

Author: ```suggestion
- [ ] Any links to companies offering support go to a page that _at least_ mentions 
  support of the project
```
